go.mod

```diff
@@ -59,6 +60,9 @@ replace (
 	github.com/openshift/api => github.com/openshift/api v0.0.0-20200331152225-585af27e34fd // release-4.5
 	github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20200326155132-2a6cd50aedd0 // release-4.5
 
+	// FIXME: Remove after https://github.com/operator-framework/api/pull/56 merges
+	github.com/operator-framework/api => github.com/njhale/api-1 v0.0.0-20200731014735-16bd162edd58
```

Comment:
https://github.com/operator-framework/api/pull/56 must merge before this can be updated

/hold